lib/cased/cli/session.rb

```diff
@@ -128,7 +128,7 @@ def initialize(reason: nil, command: nil, metadata: {}, authentication: nil)
         @authentication = authentication || Cased::CLI::Authentication.new
         @reason = reason
         @command = command || [$PROGRAM_NAME, *ARGV].join(' ')
-        @metadata = metadata
+        @metadata = metadata.merge(Cased::CLI::Config.new.metadata)
```

Comment:
This is creating a new CLI config object each time it's called and won't use the metadata that is set globally. To achieve this we reference the global config instead:

```suggestion
        @metadata = metadata.merge(Cased.config.cli.metadata)
```